### PR TITLE
add command to override player proximity voice config

### PR DESCRIPTION
**Pull request by plally**

### CODE REVIEW

File: lua/autorun/server/toggle_voice_init.lua [hunk 1]
```diff
@@ -37,15 +38,23 @@ local function canHear( listener, speaker )
     return true
 end
 
+function ProximityVoiceOverridePlayerConfig( ply, enabled ) 
+    playerConfigOverride[ply] = enabled
+    if not enabled then
+        playerConfigOverride[ply] = true
+    end
+end
```
Comment (brandonsturgeon) [inline lua/autorun/server/toggle_voice_init.lua]: Won't this always result in `true`?

File: lua/ulx/modules/sh/proximity_voice_commands.lua [hunk 2]
```diff
@@ -10,5 +10,20 @@ end
 local proximityGlobal = ulx.command( CATEGORY_NAME, "ulx enableproximityvoice", forceProximityVoice )
 proximityGlobal:addParam{ type=ULib.cmds.BoolArg, invisible=true }
 proximityGlobal:defaultAccess( ULib.ACCESS_ADMIN )
-proximityGlobal:help( "Enabled proximity voice for everyone" )
+proximityGlobal:help( "Enable proximity voice for everyone" )
 proximityGlobal:setOpposite( "ulx disableproximityvoice", {nil, true} )
+
+
+local function proximityVoiceOverride( caller, targets, enabled )
+    for _, ply in pairs( targets ) do
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/proximity_voice_commands.lua:18]:
```suggestion
    for _, ply in ipairs( targets ) do
```

File: lua/ulx/modules/sh/proximity_voice_commands.lua [hunk 3]
```diff
@@ -10,5 +10,20 @@ end
...
+        ProximityVoiceOverridePlayerConfig( ply, enabled )
+    end
+
+    local word = enabled and "disabled" or "enabled"
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/proximity_voice_commands.lua:22]:
```suggestion
    local word = enabled and "enabled" or "disabled"
```?

Comment (plally) [reply lua/ulx/modules/sh/proximity_voice_commands.lua:22]: The way I have it is correct
image

Comment (plally) [reply lua/ulx/modules/sh/proximity_voice_commands.lua:22]: yeah forgot to apply my changes from gcompute

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**